hybrid-blazor-apps.md

```diff
@@ -0,0 +1,100 @@
+
+1. Install [WebView](https://docs.microsoft.com/en-us/dotnet/maui/user-interface/controls/webview).
+
+1. Make sure you have the necessary bits to work with WinForms/WPF/MAUI apps. For MAUI installation, follow the instructions in the [official documentation](https://docs.microsoft.com/en-us/dotnet/maui/get-started/first-app?pivots=devices-android).
+
+## Add the UI for Blazor components
+
+The process for adding Telerik UI for Blazor in the WinForms/WPF/MAUI app is similar to including the components in a native Blazor app.
+
+#### 1. Get the `Telerik UI for Blazor` package
+
+To use the  UI for Blazor components you need to install the `Telerik.UI.for.Blazor` package and include its reference in the `.csproj` file of the app. [Read more on where to get the `Telerik.UI.for.Blazor` package...]({%slug getting-started/what-you-need%}#get-the-telerik-packages)
+
+#### 2. Add the Telerik client assets
+
+To have the Telerik Blazor components look and behave as expected, you need the Telerik [CSS and JavaScript assets]({%slug getting-started/what-you-need%}#client-assets). Include the assets inside the `<head>` of the `wwwroot/index.html` file.
+
+You may add the Telerik resources as [static assets]({%slug getting-started/what-you-need%}#static-assets) or reference them from a [cloud CDN]({%slug getting-started/what-you-need%}#cdn).
+
+#### 3. Include `@using` statements 
+
+You can set the project to recognize all Telerik components without explicit `@using `statements on every `.razor` file. To achieve this, add the following to your `~/_Imports.razor`file:
+
+````
+@using Telerik.Blazor
+@using Telerik.Blazor.Components
+@using Telerik.Blazor.Services
+````
+
+#### 4. Add the `TelerikRootComponent`
+
+You must add a `TelerikRootComponent` component as a top-level component in the app to make sure it wraps all the content. At the time of writing, custom layouts are not supported, so you can add it to the:
+
+* `Shared/MainLayout.razor` for MAUI apps
+* `Main.razor` for WPF and WinForms apps
+
+Make sure that the `TelerikRootComponent` matches the webview viewport. You might need to remove the default margin and padding from the body to ensure the content matches the viewport.
+
+Once custom layouts are supported, you will be able to configure a Telerik layout in the same way as with regular Blazor web apps (check [Common Configuration]({%slug getting-started/what-you-need%}#common-configuration)).
+
+#### 5. Add the UI for Blazor components
+
+After completing the above steps, you will be able to add your desired Telerik Blazor components in the app as in a native Blazor app. Explore the available UI for Blazor components and their features in our [live demos](https://demos.telerik.com/blazor-ui).
+
+
+## Run the apps
+
+You can now run the application in the expected for the technology stack way, following the corresponding instructions for how to run the app. Here are the related resources for ease of access:
```

Comment:
We mention that the developer should be familiar with the technology stack from the beginning, so this part might be unnecessary. Or we should move it up. 

```suggestion
You can now run the hybrid application. Refer to the following resources for each technology stack:
```